lib/api/actions/functions.php

```diff
@@ -410,119 +410,190 @@ function beans_reset_action( $id ) {
 }
 
 /**
- * Get action.
+ * Get the action's configuration for the given ID and status. Returns `false` if the action is not registered with
+ * Beans.
  *
+ * @since  1.0.0
+ * @since  1.5.0
  * @ignore
+ * @access private
+ *
+ * @param string $id     The action's Beans ID, a unique ID tracked within Beans for this action.
+ * @param string $status Status for which to get the action.
+ *
+ * @return array|bool
  */
 function _beans_get_action( $id, $status ) {
-
 	global $_beans_registered_actions;
 
-	$id = _beans_unique_action_id( $id );
+	$id                 = _beans_unique_action_id( $id );
+	$registered_actions = beans_get( $status, $_beans_registered_actions );
 
-	if ( ! $registered = beans_get( $status, $_beans_registered_actions ) ) {
+	// If the status is empty, return false, as no actions are registered.
+	if ( empty( $registered_actions ) ) {
 		return false;
 	}
 
-	if ( ! $action = beans_get( $id, $registered ) ) {
+	$action = beans_get( $id, $registered_actions );
+
+	// If the action is empty, return false.
+	if ( empty( $action ) ) {
 		return false;
 	}
 
 	return (array) json_decode( $action );
-
 }
 
 /**
- * Set action.
+ * Store the action's configuration for the given ID and status.
  *
+ * What happens if the action's configuration is already registered?  If the `$overwrite` flag is set to `true`,
+ * then the new action's configuration is stored, overwriting the previous one. Else, the registered action's
+ * configuration is returned.
+ *
+ * @since  1.0.0
+ * @since  1.5.0
  * @ignore
+ * @access private
+ *
+ * @param string $id        The action's Beans ID, a unique ID tracked within Beans for this action.
+ * @param array  $action    The action configuration to store.
+ * @param string $status    Status for which to store the action.
+ * @param bool   $overwrite Optional. When set to `true`, the new action's configuration is stored, overwriting a
+ *                          previously stored configuration (if one exists).
+ *
+ * @return array
  */
-function _beans_set_action( $id, $action, $status, $overwrite = false ) {
-
-	global $_beans_registered_actions;
-
+function _beans_set_action( $id, array $action, $status, $overwrite = false ) {
```

Comment:
Add a `@since 1.5.0` tag mentioning that `$action` now must be an array.

Reply:
@ThierryA This requirement has been removed in this PR per the notes above.